circus-train-integration-tests/src/test/data/com/hotels/bdp/circustrain/integration/CircusTrainS3S3IntegrationTest/partitioned-single-table-with-no-partitions-mirror.yml

```diff
@@ -8,3 +8,5 @@ table-replications:
       table-name: ct_table_p_copy
 security:
   credential-provider: jceks://file/${config-location}/aws.jceks
+copier-options:
+  s3s3copier-retry-maxattempts: 3
```

Comment:
We might want to consider renaming this to s3-copy-retry-maxattempts so we can use the same option for both the S3S3Copier and the S3MapReduceCopier (if we decide to add a similar retry feature to the latter).

Reply:
Or if we don't add it to the S3MapReduceCopier we should rename this to `s3s3-retry-max-attempts` to be more consistent with the other s3s3 specific options (e.g. `s3s3-multipart-copy-threshold-in-bytes`, `s3s3-multipart-copy-part-size-in-bytes`).

Reply:
Seeing as the map reduce copier has a retry already built in, shall we go with `s3s3-retry-max-copy-attempts` for consistency with the other options for now?

Reply:
Yes.